shim.c

```diff
@@ -1966,10 +1966,18 @@ efi_main (EFI_HANDLE passed_image_handle, EFI_SYSTEM_TABLE *passed_systab)
 die:
 		console_print(L"Something has gone seriously wrong: %s: %r\n",
 			      msgs[msg], efi_status);
+
+		/*
+		 * Provide additional information when the SBAT self check fails
+		 */
+		if ( msg == SBAT_SELF_CHECK ) {
+			console_print(L"\nThe current shim has likely been revoked. " 
+						   "Please update to a newer shim version.\n");
+		}
 #if defined(ENABLE_SHIM_DEVEL)
 		devel_egress(COLD_RESET);
 #else
-		usleep(5000000);
+		usleep(8000000);
```

Comment:
Do we need to change the delay??

Reply:
No, but I found 5 sec to short to read the messages. Another option would be to just count down or wait on user input. Need to see if we can do that nicely.

Reply:
Since the next thing we do is power down, there's really no reason not so have a delay that's plenty long enough to read the message. My instinct would be for 90 seconds or so.

That's probably more reason to sort the messages as clearly as we can above.

Reply:
Yeah sounds good to me. I increased it to 90 seconds for now.